src/commcare_cloud/commands/deploy.py

```diff
@@ -31,11 +31,14 @@ class Deploy(CommandBase):
         Argument('--commcare-rev', help="""
             The name of the commcare-hq git branch, tag, or SHA-1 commit hash to deploy.
         """, default=None),
+        Argument('--set', dest='fab_settings', help="""
+            fab settings in k1=v1,k2=v2 format to be passed down to fab 
+        """, default=None),
         shared_args.QUIET_ARG,
         shared_args.BRANCH_ARG,
     )
 
-    def run(self, args, unknown_args):
+    def run(self, args, unknown_args, color_warn=None):
```

Comment:
How is `color_warn` supplied? It looks like lines 57-61 will fail hard if it's `None`.

Reply:
Thanks this was totally me not understanding what my IDE was doing for me, fixed now